lib/tty.js

```diff
@@ -132,7 +132,7 @@ WriteStream.prototype._refreshSize = function() {
     this.emit('error', errors.errnoException(err, 'getWindowSize'));
     return;
   }
-  const [newCols, newRows] = winSize;
+  const { 0: newCols, 1: newRows } = winSize;
```

Comment:
Is this supposed to be more efficient?

Reply:
Not really (it's probably equivalent performance wise), it's to avoid calling user-mutable `Array.prototype[Symbol.iterator]` and `%ArrayIteratorPrototype%.next`.